smallboard/settings.py

```diff
@@ -233,16 +233,21 @@
     )
 
 # TODO: make this a per hunt setting
-from google_api_lib import whitelist
+from google_api_lib import sync_task
 
 if GOOGLE_API_AUTHN_INFO is not None:
-    SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_EMAILS = whitelist.get_file_user_emails(
+    SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_EMAILS = sync_task.get_file_user_emails(
         GOOGLE_DRIVE_HUNT_FOLDER_ID
     )
     logger.info(
         "Whitelisted emails: " + str(SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_EMAILS)
     )
+    GOOGLE_HUMAN_DRIVE_HUNT_FOLDER_URL = ""
+    GOOGLE_HUMAN_DRIVE_HUNT_FOLDER_URL = sync_task.get_human_drive_folder(
+        GOOGLE_DRIVE_HUNT_FOLDER_ID
+    )
 else:
+    GOOGLE_HUMAN_DRIVE_HUNT_FOLDER_URL = ""
```

Comment:
Instead of repeating `GOOGLE_HUMAN_DRIVE_HUNT_FOLDER_URL = ""`, can just initialize before the `if GOOGLE_API_AUTHN_INFO is not None` and remove it from the if and else blocks.